util/dvsim/SimCfg.py

```diff
@@ -532,39 +527,14 @@ def _create_deploy_objects(self):
         self.deploy = self.builds + self.runs
 
         # Create cov_merge and cov_report objects
-        if self.cov:
-            self.cov_merge_deploy = CovMerge(self)
+        if self.cov and not self.build_only:
```

Comment:
Also, if `self.runs` is empty, then there is nothing to run coverage on (try running `-i bogus_test --cov`).

Reply:
You're right. I've changed the test to `... and self.runs`, which should do what we want.